docs/topics/concept-getting-started-intro.adoc

```diff
@@ -2,16 +2,38 @@
 = Introduction to {ProductName}
 This section provides a brief overview of the foundational concepts behind {ProductName}.
 
-.Modern Application Development
-Modern application development involves building and deploying to a cloud environment such as OpenShift. This requires:
+.{ProductName}
+{ProductName} is:
 
-* familiarity with the target cloud environment
-* an application architecture designed to run in that cloud environment
-* design patterns that enable you to manage your services
-* developer tools that simplify the development and release cycle
+* a collection of runtimes, such as {WildFlySwarm} and {SpringBoot}, designed to run in OpenShift
+* a prescriptive approach to cloud-native development
 
-The goal of {ProductName} is to provide opinionated architectures, design patterns, tools, and best practices in ready-to-go examples that run in OpenShift in order to enable modern application development.
+The goal of {ProductName} is to provide opinionated architectures, design patterns, tools, and best practices in ready-to-go examples that run in OpenShift in order to enable cloud-native development.
 
+.Cloud-native Development
+Cloud-native development is an approach to developing, deploying, and running applications that fully embrace the advantages of cloud computing. Cloud-native development has four key tenants:
+
+DevOps::
+The processes and practices used to do cloud-native development are implemented in software, not just defined.
+
+Containers::
+Containers are used as deployment targets for your cloud-native applications.
+
+Service-based Architecture::
+Cloud-native applications use a modular and loosely-coupled architecture such as microservices.
+
+Automation::
+Cloud-native applications are deployed through pipelines using instrumentation.
```

Comment:
This seems to duplicate the "DevOps" item above. Also not sure what "using instrumentation" means.